README.md

```diff
@@ -1,7 +1,61 @@
-WHEN CLONING THIS REPO, CREATE THE DATABASE BY:
+# NC_NEWS
+NC_NEWS is a backend service for a news platform
 
-Create .env files for Test and Development, add the following database for both enviroment:
-1. For .env.test : 
-PGDATABASE=nc_news_test
-2. For .env.development:
+## Hosted Version Link
+```bash
+https://news-app-u364.onrender.com
+```
+
+## Installation
+
+This project is run on package manager: [npm](https://www.npmjs.com).
+
+1. Git Clone from
+```bash
+https://github.com/booayl/backend-project
+```
+
+2. Install Dependencies
+```bash
+npm install
+```
+
+3. Seed Local Database
+```bash
+npm run setup-dbs
+```
+
+4. To Run tests
+```bash
+npm test
+```
+
+5. Create the .env files for development:
```

Comment:
i'd probably put this before any of the other terminal commands you're expecting users to run - especially npm t  and npm run seed will have a problem if these files aren't created first! :)